config/samples/emqx/v1beta4/rebalance.yaml

```diff
@@ -3,6 +3,7 @@ kind: Rebalance
 metadata:
   name: rebalance-sample
 spec:
+  instanceKind: EmqxEnterprise
```

Comment:
This will error, because the `apps.emqx.io/v1beta4 Rebalance` didn't have this field